drivers/gps/nrf9160_gps/Kconfig

```diff
@@ -72,26 +72,27 @@ endchoice
 
 config NRF9160_GPS_SET_MAGPIO
 	bool "Let the driver set MAGPIO configuration"
-	default y if BOARD_NRF9160DK_NRF9160NS
+	default y if BOARD_NRF9160DK_NRF9160NS || BOARD_THINGY91_NRF9160NS
 
 if NRF9160_GPS_SET_MAGPIO
 
 config NRF9160_GPS_MAGPIO_STRING
 	string "MAGPIO string"
 	default "AT\%XMAGPIO=1,0,0,1,1,1565,1586" if BOARD_NRF9160DK_NRF9160NS
+	default "AT\%XMAGPIO=1,1,1,7,1,746,803,2,698,748,2,1710,2200,3,824,894,4,880,960,5,791,849,7,1565,1586" if BOARD_THINGY91_NRF9160NS
```

Comment:
Kconfig settings like this has failed in SES menuconfig before.
Has it been verified that this work as expected when using latest SES-Nordic Edition 5.10c ?

Reply:
Oh okay, I'll check and see how it works out with SES

Reply:
Using SES 5.10c, the project loads, compiles and runs nicely on Thingy:91.

Reply:
thanks for verifying.